src/App.tsx

```diff
@@ -132,91 +142,150 @@ useEffect(() => {
   }, [hoveringId]);
 
   const handleflyTransitionEnd = () => {
-    const element = document.getElementById('earth_orbits_container');
+    const element = document.getElementById("earth_orbits_container");
     if (element) {
-      if(element.classList.contains('earth_fly_out')) { 
+      if (element.classList.contains("earth_fly_out")) {
         //this is handling the fly out
-        element.classList.remove('earth_fly_out');
+        element.classList.remove("earth_fly_out");
       } else {
         //this is handling the fly in
         setflyTransitionEnded(true);
       }
     }
-
   };
 
-  const handleSetStaticImg = (imgId : string) => {
+  const handleSetStaticImg = (imgId: string) => {
     setCurrentStaticImg(imgId);
   };
 
   const handleReturntoMain = () => {
-    setActiveId('');
-    setCurrentStaticImg('');
-    setHoveringId('');
-    setflyToId('');
+    setActiveId("");
+    setCurrentStaticImg("");
+    setHoveringId("");
+    setflyToId("");
     setflyTransitionEnded(false);
     adjustElementScale(document.documentElement.clientWidth);
-  }
+  };
 
   return (
+    <div className="background_container">
+      <div id="splash_screen_background" className="splash_screen_background">
+        <SplashScreen setSplashScreen={handleSetSplashScreen} />
+      </div>
+
+      {!isSplashScreen && (
+        <div className="nav_container">
+          {!flyToId && (
+            <div className="orbit_title_container">
+              {orbitIds.map((id, index) => {
+                return (
+                  <OrbitTitleGroup
+                    id={id}
+                    setActive={handleSetActive}
+                    activeId={activeId}
+                    setHover={handleSetHover}
+                    hoverId={hoveringId}
+                    tabIndex={index}
+                  />
+                );
+              })}
+
+              <div className="popup_line_container">
+                {orbitIds.map((id) => {
+                  return <PopUpLine id={id} activeId={activeId} />;
+                })}
+              </div>
```

Comment:
I added this div to contain the popup lines.
This doesn't seem to affect 'your domain'.
I used it to help align the popup lines container to the earth orbit container so the connecting line will land consistently.